examples/spiral/spiral.py

```diff
@@ -0,0 +1,30 @@
+import sys
+import scipy
+from basic_api import write_samples_to_file
+from accumulator import SampleAccumulator
+
+
+def build_poly(side_count, corner_radius):
+    accumulator = SampleAccumulator({'x': 0.0, 'y': 0.0},
+                                     0.0,
+                                     {'line_sample_count': 2,
+                                     'arc_sample_count': 20})
+    for counter in range(side_count):
+        accumulator.add_sample('arc', {'radius':
+        float(corner_radius*float(side_count - counter + 1.)/float(side_count + 1.)),
+                                       'central_angle':
+                                       scipy.pi*2.*0.05,
```

Comment:
The reference to `scipy.py*2.*0.05` is a magic number? Should this be specified in `run.sh` rather than hardcoded in the python file?